src/Components/Publishing/Layouts/StandardLayout.tsx

```diff
@@ -177,3 +197,7 @@ const FooterContainer = styled.div`
     margin: 0 20px;
   `};
 `
+
+const TrackerImage = styled.img`
+  display: none;
```

Comment:
if you use `display: none;` does it still track?

Reply:
Yeah it still makes the request as long as width and height aren't set to 0

Reply:
Actually apparently even that is fine https://stackoverflow.com/a/18448053